cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+"""This module tests Nuage EMS events."""
+import pytest
+from wrapanapi.utils.random import random_name
+
+from cfme.networks.provider.nuage import NuageProvider
+from cfme.utils.log import logger
+
+
+pytestmark = [
+    pytest.mark.provider([NuageProvider])
+]
+
+
+@pytest.yield_fixture
+def with_nuage_sandbox(networks_provider):
+    nuage = networks_provider.mgmt
+    sandbox = box = {}
+
+    # Create empty enterprise aka 'sandbox'.
+    enterprise = box['enterprise'] = nuage.create_enterprise()
+    logger.info('Created sandbox enterprise {} ({})'.format(enterprise.name, enterprise.id))
+
+    # Fill the sandbox with some entities.
+    # Method `create_child` returns a tuple (object, connection) and we only need object.
+    box['template'] = enterprise.create_child(
+        nuage.vspk.NUDomainTemplate(name=random_name()))[0]
+    box['domain'] = enterprise.create_child(
+        nuage.vspk.NUDomain(name=random_name(), template_id=box['template'].id))[0]
+    box['zone'] = box['domain'].create_child(
+        nuage.vspk.NUZone(name=random_name()))[0]
+    box['subnet'] = box['zone'].create_child(
+        nuage.vspk.NUSubnet(
+            name=random_name(),
+            address='192.168.0.0',
```

Comment:
Q: shouldn't such data be stored in cfme_data yaml ?
as far as I understand it is provider related.